src/slack.ts

```diff
@@ -128,14 +130,16 @@ export interface ConfigOptions {
 }
 
 export async function send(
-  url: string,
   jobName: string,
   jobStatus: string,
   jobSteps: object,
+  url?: string,
+  token?: string,
   channel?: string,
+  channel_id?: string,
   message?: string,
   opts?: ConfigOptions
-): Promise<IncomingWebhookResult> {
+): Promise<Object> {
```

Comment:
This makes type safety pointless if you just accept an Object. What is the actual response type?